src/test/java/io/seqera/tower/cli/BaseCmdTest.java

```diff
@@ -55,6 +55,15 @@
         }
     }
 
+    protected byte[] loadResource(String name, String ext) {
+        try {
+            return this.getClass().getResourceAsStream("/runcmd/" + name + "." + ext).readAllBytes();
```

Comment:
I am not very sure about the cli runtime but shouldn't this stream be closed?
Using the try-with resources syntax it should be a simple change